ci-operator/config/openshift/odo/openshift-odo-master__v4.1.yaml

```diff
@@ -16,23 +16,11 @@ tag_specification:
 binary_build_commands: make bin
 
 tests:
-- artifact_dir: /tmp/artifacts
-  as: unit
-  commands: make openshiftci-presubmit-unittests
-  container:
-    from: src
 - artifacts_dir: /tmp/artifacts
-  as: integration
+  as: integration-e2e-benchmark
```

Comment:
can we use some shorter name? Just `tests` might be enought

Reply:
```tests``` looks more generic, less talkative and the name ```integration-e2e-benchmark``` is more talkative and itself suggest what we test.

Reply:
You are right that `tests`, might be a little too generic. but `integration-e2e-benchmark` is really talkative and that is the issue. There is just no reason to have those long names.

Reply:
I can't think of a better name. So if you like the name keep it.